deps.bzl

```diff
@@ -1347,6 +1365,12 @@ def prysm_deps():
         sum = "h1:UImYN5qQ8tuGpGE16ZmjvcTtTw24zw1QAp/SlnNrZhI=",
         version = "v1.9.5",
     )
+    go_repository(
+        name = "com_github_grpc_ecosystem_grpc_gateway_v2",
+        importpath = "github.com/grpc-ecosystem/grpc-gateway/v2",
```

Comment:
Will there be any side effects to this getting bumped?

Reply:
Good question... this adds a new indirect dependency, but does not touch the `github.com/grpc-ecosystem/grpc-gateway` dep. So I think it is fine.

@rkapka is grpc gateway still used these days? I think it's all new http API server code for the beacon API

Reply:
grpc gateway i don't think is really used, I think there's some protobuf thing and something for metrics but not the endpoints themselves, I removed all that